.travis.yml

```diff
@@ -63,9 +60,9 @@ matrix:
     - python: 3.5
 
 before_install:
-  - wget -O geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.12.0/geckodriver-v0.12.0-linux64.tar.gz
-  - gunzip -c geckodriver.tar.gz | tar xopf -
-  - chmod +x geckodriver && sudo mv geckodriver /usr/local/bin
+  - wget -O /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.12.0/geckodriver-v0.12.0-linux64.tar.gz
```

Comment:
Updating should be in a separate commit. I'd also argue that using cache should be taken care of separately.